modules/overrides/userconfigurable/client/limits.go

```diff
@@ -25,9 +29,48 @@ func (l *Limits) GetMetricsGenerator() *LimitsMetricsGenerator {
 	return nil
 }
 
+type Duration struct {
+	time.Duration
+}
```

Comment:
Mostly a nitpick: I think it would be cleaner to move the `Duration` struct into a separate file:
- this file (unfortunately) already has a lot of boilerplate making it difficult to read
- keeping it in a separate file will make it easier to reuse later, we can just move the file to shared package when we need it

What do you think of moving this struct and the corresponding test functions into `duration.go`?

Reply:
Makes sense to me. I moved the code. Thanks!